app/models/user.rb

```diff
@@ -5,4 +5,6 @@ class User < ApplicationRecord
   include Identifiable
   include Named
   include Privileged # depends on Eventable
+
+  has_many :subscriptions, class_name: "Hackathon::Subscription", foreign_key: "subscriber_id", inverse_of: :subscriber, dependent: :destroy
```

Comment:
I don't want this to become a god object - could we do this some other way?

Reply:
This association's perfectly fine to have. In my opinion, it's unavoidable since it's one of the primary models in the codebase.

If we removed this, we'd be writing the same code in other (ugly) ways:

```ruby
Hackathon::Subscription.create(user: @user, ...)
```
rather than 
```ruby
@user.subscriptions.create!(...)
```

God classes are a valid concern, but it should prevent us from adding this association

Reply:
By default, `subscriber` is the `Current.user`

Reply:
It appears that `foreign_key: "subscriber_id"` is necessary. Otherwise, it looks for a `user_id` column.